src/components/light.js

```diff
@@ -255,8 +255,9 @@ module.exports.Component = registerComponent('light', {
     }
   },
 
-  onSetTarget: function (targetEl) {
-    this.light.target = targetEl.object3D;
+  onSetTarget: function (targetEl, light) {
+    if (this.light) { this.light.target = targetEl.object3D; }
```

Comment:
Can we just rely on the `light` parameter? Do we need to check both this.light and light?
````javascript
light.target = targrtEl.object3D
````